contracts/cw-core/src/contract.rs

```diff
@@ -850,6 +872,22 @@ pub fn migrate(deps: DepsMut, _env: Env, msg: MigrateMsg) -> Result<Response, Co
                     PROPOSAL_MODULES.save(deps.storage, address, proposal_module)?;
                     Ok(())
                 })?;
+
+            // Update config to have the V2 "dao_uri" field.
+            let config_item: Item<V1Config> = Item::new("config");
```

Comment:
@blue-note - in previous PRs we've made a new storage key `config_v2` and stored the new value there, keeping the old one. I think overriding is a fine approach, curious if there is a reason we did that here and not elsewhere?

Reply:
oh yeah good point -- I also don't think technically it is a problem to override, but might be better to create a new namespace just to be safe